### DISCUSSION TITLE
Welcome to TutorTrek Discussions!

### DISCUSSION THREAD

**Discussion by abinth11** [abinth11/TutorTrek#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JavascriptDev347** [comment]
Hi bro 
I tried to use your TutorTreck project but it gives me an error, can you help me?
![image](https://github.com/abinth11/TutorTrek/assets/99870226/85d2d92d-198e-457d-ae71-f152348d859f)
